src/azure-cli/azure/cli/command_modules/acs/_params.py

```diff
@@ -221,6 +221,7 @@ def load_arguments(self, _):
         c.argument('workspace_resource_id')
         c.argument('enable_msi_auth_for_monitoring', arg_type=get_three_state_flag(), is_preview=True)
         c.argument('enable_syslog', arg_type=get_three_state_flag(), is_preview=True)
+        c.argument('data_collection_settings', options_list=['--data-collection-settings'], is_preview=True)
```

Comment:
`--data-collection-settings` is the default parameter name and does not need to be defined repeatedly
```suggestion
        c.argument('data_collection_settings', is_preview=True)
```